Lombiq.HelpfulLibraries/Libraries/Contents/ContentExtensions.cs

```diff
@@ -164,5 +164,14 @@ public static async Task SanitizeContentItemVersionsAsync(this IContent content,
             content.ContentItem.PublishedUtc ??
             content.ContentItem.CreatedUtc ??
             DateTime.MinValue;
+
+        /// <summary>
+        /// Indicates whether the <see cref="ContentItem"/> is a newly created one or an already existing one.

```

Comment:
It's not clear if we're talking about a newly instantiated content item (i.e. `IContentManager.NewAsync()`) or a newly created one (`CreateAsync()`).